source/jobs/recover-jobs.txt

```diff
@@ -0,0 +1,77 @@
+.. _rm-sync-jobs-recoverability:
+
+==================
+Recover a Sync Job
+==================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Relational Migrator can attempt to recover continuous (CDC) sync jobs 
```

Comment:
(nit)
I think "can attempt" is too tentative. Maybe just attempts? 
If we go with "can attempt" we should explain the circumstances under which the attempt can be made.

Reply:
I think that can attempt in this content refers to the fact that the recovery can be manual or automatic. It can attempt in the sense that you can go and click retry type of thing. If we put just "attempts" it makes it sound like RM will recover your failed jobs all the time which is not the case. We do allude to context of when the recoveries are attempted in the steps under the tabs.